x-pack/plugins/canvas/public/components/render_with_fn/render_with_fn.js

```diff
@@ -25,7 +25,7 @@ export class RenderWithFn extends React.Component {
       destroy: PropTypes.func.isRequired,
       onDestroy: PropTypes.func.isRequired,
     }),
-    config: PropTypes.object.isRequired,
+    config: PropTypes.object,
```

Comment:
Seems like adding a `defaultProp` for this, so its value is always an object, would be worthwhile. Some of the child components expect to get an object, and passing in `null` is going cause problem. The same is probably true of the `payload` prop in the `Debug` component. 

But I would like to know more about why the value is now `null` instead of an empty object. Maybe I missed that in the grammar change here... and I wonder if the grammar should always generate an object, even for an empty expression. Basically, `assert.deepStrictEqual(fromExpression(''), {})` instead of `assert.strictEqual(fromExpression(''), null)`

Reply:
Yes, your comments are on the mark, see my "something else" part [here](https://github.com/elastic/kibana/pull/28796#issuecomment-478189303)